core/src/main/java/hudson/PluginManager.java

```diff
@@ -2225,7 +2225,7 @@ public JSONArray doPrevalidateConfig(StaplerRequest req) throws IOException {
     @RequirePOST
     public HttpResponse doInstallNecessaryPlugins(StaplerRequest req) throws IOException {
         prevalidateConfig(req.getInputStream());
-        return HttpResponses.redirectViaContextPath("updates/");
+        return HttpResponses.redirectViaContextPath("updateCenter");
```

Comment:
This could be changed to `pluginManager/updates/` if it is important to include the `pluginManager` portion in the path of the redirect.  Both `/updateCenter` and `/pluginManager/updates/` exist.
```suggestion
        return HttpResponses.redirectViaContextPath("updateCenter");
```

Reply:
this url is the more appropriate url to use (pluginManager/updates/)

As the updateCenter page shouldn't be linked / used anymore as far as I know